src/components/BodyRouter.js

```diff
@@ -21,18 +25,23 @@ function BodyRouter(props) {
                         return <History />
                     default:
                         return context.user && <>
-                            <div>
-                                <RoutingButton text="Start a ranked climb" routeTarget={'rankedClimb'} />
-                                <RoutingButton text="Start a casual climb" routeTarget={'casualClimb'} />
-                                <RoutingButton text="Time a hangboard hang" routeTarget={'hangboardTimer'} />
+                            <div className='btnGroup01'>
```

Comment:
This class has no CSS associated with it; implement or remove

Reply:
In the general case, if you're setting things up as a series of things (like group 1, group 2), prefer a grid/stack to organise them for a more modular layout. Consider implementing a stack instead around the two inner stacks.

Reply:
Removed div wrapper.